core/src/main/java/io/neow3j/protocol/core/JsonRpc2_0Neow3j.java

```diff
@@ -941,9 +1094,22 @@ public JsonRpc2_0Neow3j(
      */
     @Override
     public Request<?, NeoGetApplicationLog> getApplicationLog(String txId) {
+        return getApplicationLog(new Hash256(txId));
+    }
+
+    /**
+     * Gets the contract event information based on the specified transaction
+     * id. The contract event information is stored under the ApplicationLogs
+     * directory.
```

Comment:
```suggestion
     * Gets the application logs of the specified transaction.
```

Apply the same change to the javadoc of the method above.